<p>Avoid using try-catch-finally</p>
<h2>Noncompliant Code Example</h2>
<pre>
		try{
            ...
        }catch{
            ...
        }finally{
            ...
        }
</pre>
<h2>Compliant Solution</h2>
<pre>
		try{
            ...
        }catch{
            ...
        }
</pre>
