<p>
    Using try catch without finally
</p>
<h2> Noncompliant Code Example</h2>
<p>
    try{
        ...
    }catch(){
        ...
    }finaly{

    }
</p>
<h2> Compliant Code Example</h2>
<p>
    try{
    ...
    }catch(){
    ...
    }
</p>